packages/cli/generation/ir-generator-tests/src/dynamic-snippets/__test__/test-definitions/file-upload.json

```diff
@@ -514,7 +514,7 @@
                         {
                             "type": "fileArray",
                             "name": {
-                                "originalName": "fileList",
+                                "originalName": "file_list",
```

Comment:
Ahh this explains the dynamic snippets failure - would you mind updating the `DynamicSnippetsGenerator.test.ts` case so that the provided parameter matches? Updating the wire name changes the behavior.